### ISSUE TITLE
Emulator doesn’t allow clickable response

### ISSUE THREAD

**Issue by ganeshnj** [Microsoft/BotBuilder#469]
I‘ve just updated to the newer version (1.0.1.70) of Bot Emulator and since then I’m not able to respond by clicking on buttons (E.g. Prompt responses) in the conversation. The emulator does nothing on click, however text response still working.

Anyone else facing the same issue?

**Comment by tomlm** [comment]
Can you send example serialization of your message?  I am not able to repro this

**Comment by ganeshnj** [comment]
Today, when I opened the Emulator, it restored itself to version 1.0.0.66 and now it works fine.

**Comment by JonathanHuss** [comment]
Hi Tom...I'm from Microsoft as well and can repro the issue.  If it's helpful for you, I'd be happy to sync with you on it.

**Comment by ganeshnj** [comment]
The newer version (1.0.1.70) still has the same issue.

**Comment by tomlm** [comment]
I still can't repro this, you will need to provide more information.  What does your button definition look like?

**Comment by tomlm** [comment]
OK, I have figured out what is going on here.  The emulator correctly implements buttons that have {"title": "xyz", "value":"qwe" }.  The C# Bot Builder SDK on the other hand generates buttons with just the "Title" property and no "Value", which causes the emulator to think there is no action to perform.  When I tried to repro I was using a test written below the bot builder SDK, and so I couldn't repro it.

We are going to push out a new build of the emulator today.
